code-challenges.js

```diff
@@ -10,9 +10,20 @@ var people = [
 ]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
 
-
-
-
+// Okay, this challenge seems very hard. I know we did a similar challenge for our week one assessment where we would access values in an array of objects.
+// // We will start by creating a function, and we already know that we will need to use a map in order to access the values in our array of objects and capitalized the names.
+const objects = (array) => {
+  let firstLetterCapitalized = array.map(value => {
+// // Now, because each name has a first and a last name (and there is no comma so it's only one object) we have to split our array in order to map through it, and capitalized both the firt and last name.
+let splitThoseNames = value.name.split(" ")
+// console.log(splitThoseNames);
+// Now that we have splitted our name to : [ford, prefect], we have to map over our local variable and capitalize the first letter at index [0] and then use substring. Lastly we want to attach the rest of the sentence.
+ return splitThoseNames.map(value => value.charAt(0).toUpperCase() + value.substring(1)).join(" ") + ` is a ${value.occupation}.`
+})
+// return the whole thing!
+return firstLetterCapitalized
```

Comment:
I found this logic to straight forward and effective. I personally struggled to try to access the values in the name object space. Your code is very viewer-friendly and understandable. Your variable names are very helpful as well.